polars/11_missing_data.py

```diff
@@ -0,0 +1,190 @@
+# /// script
+# requires-python = ">=3.12"
+# dependencies = [
+#     "marimo",
+#     "polars==1.22.0",
+# ]
+# ///
+
+import marimo
+
+__generated_with = "0.11.13"
+app = marimo.App(width="medium")
+
+
+@app.cell
+def _():
+    import marimo as mo
+    return (mo,)
+
+
+@app.cell(hide_code=True)
+def _(mo):
+    mo.md(
+        """
+        # Handling Missing Data in Polars
+
+        _By Felix Najera._  
+
+        In real‑world datasets, missing values are inevitable. Polars offers a rich set of methods to detect, remove, and impute missing data efficiently. This notebook walks through common patterns and best practices for dealing with nulls in Polars DataFrames.
+        """
+    )
+    return
+
+
+@app.cell
+def _(mo):
+    mo.accordion({
+        "### TLDR":"""You can identify and manage missing values using specific methods: [```.is_null```](https://docs.pola.rs/api/python/stable/reference/expressions/api/polars.Expr.is_null.html) to check for missing values, [```fill_null```](https://docs.pola.rs/api/python/stable/reference/expressions/api/polars.Expr.fill_null.html) replaces them with specified values, and [`drop`](https://docs.pola.rs/api/python/stable/reference/dataframe/api/polars.DataFrame.drop.html) removes rows or columns containing null values, applicable to all data types including strings. Taking special care of variables with String cases see the Strings notebook for more."""
+
+    })
+    return
+
+
+@app.cell
+def _(mo):
+    mo.md("""Polars datatype specific features for missing data are NaN(for float values) and null(for everything else).""")
```

Comment:
`null` and `NaN` are completely different things in polars, Not a Number is not a missing value in polars, it is a float value like `0.0` or `inf`
There are some methods dedicated specially to dealing with it like `expr.is_nan()` (similar to but distinct from `expr.is_null()`), and you can have both NaN and nulls in float columns.

That confusion might arise from the way NaN is used by other libraries like numpy and polars?